### PR TITLE
Updates to the README file

### PR DESCRIPTION
**Pull request by erikaheidi**
This PR brings updates to the README file showing a practical usage example of cloning a repository locally, and updating the format so it looks more like the template README.

### CODE REVIEW

**Review by jdolitsky — APPROVED** [review]

**Review by amouat — APPROVED** [review]

**Comment by amouat** [comment]
I didn't have to do chmod thing on Mac, but I'm not sure why tbh...

File: README.md [hunk 1]
```diff
@@ -1,8 +1,108 @@
...
+```
+You should get output similar to this:
+
+```
+git version 2.37.1
+```
+
+### Cloning a Repository Locally
+
+Because your local system user's UUID might differ from that of the container image, if you want to clone repositories locally using this image you'll need to set up special permissions for the target dir. Then, you'll be able to set up a volume and have the contents of the cloned repo replicated on your host machine.
```
Comment (imjasonh) [inline README.md:35]: UUID -> UID (or user ID) right?

Comment (amouat) [reply README.md:35]: Yes, good catch...

Comment (amouat) [reply README.md:35]: fixed in #16

**Pull request merged**